grails-app/controllers/com/unifina/controller/api/KeyApiController.groovy

```diff
@@ -174,19 +186,19 @@ class KeyApiController {
 					Permission.Operation operation = Permission.Operation.fromString(permission)
 					SecUser user = request.apiUser
 					boolean logIfDenied = false
-					if (operation == Permission.Operation.READ) {
-						if (!hasPermission(Permission.Operation.READ, k.getPermissions())) {
-							permissionService.grant(user, res, k, Permission.Operation.READ, logIfDenied)
+					if (operation == Permission.Operation.STREAM_GET) {
```

Comment:
I guess we have to push through with this. 

`operation` from the frontend can be `STREAM_PUBLISH` or `STREAM_SUBSCRIBE`, as established above.

`if (operation == STREAM_SUBSCRIBE)` should grant (if not already granted) `STREAM_GET` and `STREAM_SUBSCRIBE`